appveyor.yml

```diff
@@ -15,8 +10,6 @@ environment:
     - nodejs_version: "6"
 
 install:
-  - npm -g install npm@latest
```

Comment:
@phated The reason of removing this line is that the latest npm don’t support old versions of nodejs (< v4.x). This is needed if we are going to support gulp-cli on these old versions and use Appveyor for test on Windows.

Or we might be able to skip test on Windows about old versions of nodejs.